scipy/stats/morestats.py

```diff
@@ -2132,21 +2202,39 @@ def ansari(x, y):
     are different.
 
     >>> ansari(x1, x2)
-    AnsariResult(statistic=511.0, pvalue=0.35506083719834347)
+    AnsariResult(statistic=511.0, pvalue=0.3550608509212331)
 
     With a p-value of 0.355, we cannot conclude that there is a
     significant difference in the scales (as expected).
 
     Now apply the test to `x1` and `x3`:
 
     >>> ansari(x1, x3)
-    AnsariResult(statistic=452.0, pvalue=0.006280278681971285)
+    AnsariResult(statistic=452.0, pvalue=0.006280279118023228)
 
     With a p-value of 0.00628, the test provides strong evidence that
     the scales of the distributions from which the samples were drawn
     are not equal.
 
+    We can use the ``alternative`` parameter to perform one-tailed test.
+    In the above example, the scale of `x1` is greater than `x3` and so
+    the ratio of scales of `x1` and `x3` is greater than 1. This means
+    that the p-value when ``alternative='greater'`` should be near 0 and
+    hence we should be able to reject the null hypothesis:
+
+    >>> ansari(x1, x3, alternative='greater')
+    AnsariResult(statistic=452.0, pvalue=0.003140139559011614)
+
+    As we can see, the p-value is indeed quite low. Use of
+    ``alternative='less'`` should thus yield a large p-value:
+
+    >>> ansari(x1, x3, alternative='less')
+    AnsariResult(statistic=452.0, pvalue=0.9971491079190463)
```

Comment:
```suggestion
    AnsariResult(statistic=452.0, pvalue=0.003140139559011614)  # random

    As we can see, the p-value is indeed quite low. Use of
    ``alternative='less'`` should thus yield a large p-value:

    >>> ansari(x1, x3, alternative='less')
    AnsariResult(statistic=452.0, pvalue=0.9971491079190463)  # random
```

Reply:
I don't think it should vary on different platforms. Does it vary for you @tupui?

Reply:
No right now it's fine, but I have an open PR which removes the `seed` from above, hence this suggestion. This is optional here. Just wanted to make my life easier for after ;)

Reply:
Oh, that makes sense! I will change this. Thanks!

Reply:
Saying this in case you don't know: in the file section, you can batch some suggestions and make a single commit if you want.

Reply:
Thanks, BTW if you have some time to review my PR it would be nice.

Reply:
> Saying this in case you don't know: in the file section, you can batch some suggestions and make a single commit if you want. :smiley:

Yeah, it's very convinient!

Reply:
> Thanks, BTW if you have some time to review my PR it would be nice.

Sure! :blush:

Reply:
Thanks I appreciate. I am starting to accumulate stalled PR waiting for a someone to take a look 😅